src/main/java/io/strimzi/kafka/bridge/http/converter/HttpBinaryMessageConverter.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http.converter;
+
+import io.strimzi.kafka.bridge.converter.MessageConverter;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
+import io.vertx.kafka.client.consumer.KafkaConsumerRecords;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+
+import javax.xml.bind.DatatypeConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HttpBinaryMessageConverter implements MessageConverter<byte[], byte[], Buffer, Buffer> {
+
+
+    @Override
+    public KafkaProducerRecord<byte[], byte[]> toKafkaRecord(String kafkaTopic, Integer partition, Buffer message) {
+
+        Integer partitionFromBody = null;
+        byte[] key = null, value = null;
```

Comment:
One declaration per line.

Reply:
why?

Reply:
Because this form is hardly ever used in idiomatic java. (I honestly don't recall the last time I saw this used). It make it harder to spot where value is declared, because people are so used to the single-declaration-per-line form. So why are you using it here?

Reply:
Because embedded developers don't have any problem on understanding and reading this line as Java ones :-) I will change if you want.

Reply:
Yeah, I know. And FWIW if this was C I wouldn't complain. But it's Java so we need to write code which is easy for typical Java devs to read, so I'd be grateful if you could change it.